src/main/java/com/example/books/model/Book.java

```diff
@@ -14,31 +14,24 @@
 
 @Data
 @Entity
-@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
-@Where(clause = "is_deleted=false")
+@SQLDelete(sql = "UPDATE books SET is_deleted = TRUE WHERE id = ?")
+@Where(clause = "is_deleted=FALSE")
 @Table(name = "books")
 public class Book {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-
     @NotNull

```

Comment:
Validation constraints should validate incoming information so used in DTOs
In models we don't use them